modules/button/react/lib/OutlineButton.tsx

```diff
@@ -83,6 +83,8 @@ export const getOutlineButtonColors = (
     canvas: {
       palette: {primary: themePrimary},
     },
+    // TODO: Remove this support after AB test is complete
+    ABTest: {secondaryOutlineButton: ABTestColors},
```

Comment:
```suggestion
    ABTest: {secondaryOutlineButton: ABTestColors} = {},
```

Reply:
Typescript doesn't like this. Your other suggestion should do the trick anyway 👍

Reply:
Unfortunately it isn't as easy as this. Typescript isn't happy with it because `{}` isn't a valid default

Reply:
This or something like it is still needed for runtime.